tests/sentry/api/endpoints/test_project_metrics_extraction_rules.py

```diff
@@ -409,12 +409,12 @@ def test_get_pagination(self):
         json_payload = {
             "metricsExtractionRules": [
                 {
-                    "spanAttribute": f"count_clicks_{i:02d}",
+                    "spanAttribute": f"count_clicks_{i:04d}",
                     "type": "c",
                     "unit": "none",
                     "tags": ["tag1", "tag2", "tag3"],
                 }
-                for i in range(0, 60)
+                for i in range(0, 2050)
```

Comment:
can you check if there is a noticeable performance impact of this change. Our tests are anyways slow so i would like to avoid this getting even slower

Reply:
There is, I measured locally a few times, there is around 0.5s increase in test execution time (from 9.3s to 9.8s), but the main reason for that is not due to this iteration, but due to the fact that we are now retuning 1000 items instead of 25 from the same API endpoint, so the fetching takes more time.

I have tried this loop with the test case when there is default per page limit of 25 and there is no significant difference in execution time.

Reply:
ok! thanks!